Update tsdb comparison doc (#2601)
apache/iotdb: docs/UserGuide/Comparison/TSDB-Comparison.md

* Read:

10 clients read data concurrently. The number of storage group is 10. There are 10 devices and each device has 10 measurements (i.e.,, 100 time series total).
The data type is *double*, encoding type is *GORILLA*

The IoTDB version is v0.11.1.

**Write performance**:

* batch size:

10 clients write data concurrently.
IoTDB uses batch insertion API and the batch size is distributed from 1ms to 1min (write N data points per write API call).

The write throughput (points/second) is:

![Batch Size with Write Throughput (points/second)](https://user-images.githubusercontent.com/24886743/106254214-6cacbe80-6253-11eb-8532-d6a1829f8f66.png)
<span id = "exp1"> <center>Figure 1. Batch Size with Write throughput (points/second) IoTDB v0.11.1</center></span>

The write delay (ms) is:

![Batch Size with Write Delay (ms)](https://user-images.githubusercontent.com/24886743/106251391-df1b9f80-624f-11eb-9f1f-66823839acba.png)
<center>Figure 2. Batch Size with Write Delay (ms) IoTDB v0.11.1</center>

* client num:

> client num is distributed from 1 to 50.

Reviewer: ```suggestion
The client num is distributed from 1 to 50.
```

![Aggregation query](https://user-images.githubusercontent.com/24886743/106251336-cf03c000-624f-11eb-8395-de5e349f47b5.png)
<center>Figure 6. Aggregation query time cost(ms) IoTDB v0.11.1</center>

![Downsampling query](Query.pnghttps://user-images.githubusercontent.com/24886743/106251353-d32fdd80-624f-11eb-80c1-fdb4197939fe.png)
<center>Figure 7. Downsampling query time cost(ms) IoTDB v0.11.1</center>

![Latest query](https://user-images.githubusercontent.com/24886743/106251369-d7f49180-624f-11eb-9d19-fc7341582b90.png)
<center>Figure 8. Latest query time cost(ms) IoTDB v0.11.1</center>

We can see that IoTDB outperforms others. 

#### More details

We provide a benchmarking tool, called IoTDB-benchamrk (https://github.com/thulab/iotdb-benchmark, you may have to use the dev branch to compile it),
it supports IoTDB, InfluxDB, KairosDB, TimescaleDB, OpenTSDB. We have an [article](https://arxiv.org/abs/1901.08304) for comparing these systems using the benchmark tool.
When we publish the article, IoTDB just entered Apache incubator, so we deleted the performance of IoTDB in that article. But after comparison, some results are presented here.

> All TSDB run on the same server one by one. 

Reviewer: ```suggestion
```

#### quick review

* Write:

> We test write performance from two aspects: *batch size* and *client num*, The number of storage group is 10. There are 1000 devices and each device has 100 measurements(i.e.,, 100K time series totally).

Reviewer: ```suggestion
We test the performance of writing from two aspects: *batch size* and *client num*. The number of storage group is 10. There are 1000 devices and each device has 100 measurements(i.e.,, 100K time series total).
```

* Read:

> 10 clients read data concurrently. The number of storage group is 10. There are 10 devices and each device has 10 measurements(i.e.,, 100 time series totally).

Reviewer: ```suggestion
10 clients read data concurrently. The number of storage group is 10. There are 10 devices and each device has 10 measurements (i.e.,, 100 time series total).
```

The data type is *double*, encoding type is *GORILLA*

> IoTDB is v0.11.1.

Reviewer: ```suggestion
The IoTDB version is v0.11.1.
```

**Write performance**:

* batch size:

10 clients write data concurrently.

> IoTDB uses batch insertion API and the batch size is distributed from 1(1ms) to 6000(1min) (write N data points per write API call).

Reviewer: ```suggestion
IoTDB uses batch insertion API and the batch size is distributed from 1ms to 1min (write N data points per write API call).
```

> it supports IoTDB, InfluxDB, KairosDB, TimescaleDB, OpenTSDB. We have a [article](https://arxiv.org/abs/1901.08304) for comparing these systems using the benchmark tool.

Reviewer: ```suggestion
it supports IoTDB, InfluxDB, KairosDB, TimescaleDB, OpenTSDB. We have an [article](https://arxiv.org/abs/1901.08304) for comparing these systems using the benchmark tool.
```

When we publish the article, IoTDB just entered Apache incubator, so we deleted the performance of IoTDB in that article. But after comparison, some results are presented here.

All TSDB run on the same server one by one. 

> - For InfluxDB, we set the cache-max-memory-size  and max-series-perbase as unlimited (otherwise it will be timeout quickly)

Reviewer: ```suggestion
- For InfluxDB, we set the cache-max-memory-size and max-series-perbase as unlimited (otherwise it will be timeout quickly).
```
